factory/usr/lib/systemd/system-generators/journald-console

```diff
@@ -2,7 +2,7 @@
 
 set -eu
 
-if [ "$(/usr/libexec/core/get-arg snapd_recovery_mode || true)" != install ]; then
+if [ "$(/usr/libexec/core/get-arg snapd_recovery_mode || true)" == run ]; then
```

Comment:
can we change this check to the same check that the-modeenv does, as in `
mode="$(/usr/libexec/core/get-mode mode)" || mode="$(/usr/libexec/core/get-arg snapd_recovery_mode)" || mode="unknown"` ?

which i guess gets the right mode both at initrd startup; and also at mid-flight reload?